app/controllers/reviews_controller.rb

```diff
@@ -3,18 +3,14 @@ class ReviewsController < ApplicationController
 
   def create
     @video = Video.find(params[:video_id])
-    review = @video.reviews.create(review_params.merge!(user: current_user))
-    return redirect_to @video
+    review = @video.reviews.build(params.require(:review).permit(:content, :rating).merge!(user: current_user))
```

Comment:
I would pass `params.require(:review).permit(:content, :rating).merge!(user: current_user)` to a method `review_params` - only because the line is already too long and its easier to read when the logic is split a bit.